src/sip/transp.c

```diff
@@ -337,6 +337,12 @@ static void sip_recv(struct sip *sip, const struct sip_msg *msg,
 		}
 	}
 
+	/* check consistency between CSeq method and that of request line */
+	if (pl_casecmp(&(msg->cseq.met), &(msg->met))){
+		(void)sip_reply(sip, msg, 400, "Bad Request");
+		return;
+	}
+
```

Comment:
is the check valid for Request or Response? or both ?

Reply:
For Request. Thanks for the hint!